sentry-android-core/src/main/java/io/sentry/android/core/SystemEventsBreadcrumbsIntegration.java

```diff
@@ -206,10 +216,43 @@ public void onReceive(final Context context, final @NotNull Intent intent) {
                   scopes.addBreadcrumb(breadcrumb, hint);
                 });
       } catch (Throwable t) {
-        options
-            .getLogger()
-            .log(SentryLevel.ERROR, t, "Failed to submit system event breadcrumb action.");
+        // ignored
+      }
+    }
+
+    // in theory this should be ThreadLocal, but we won't have more than 1 thread accessing it,
+    // so we save some memory here and CPU cycles. 64 is because all intent actions we subscribe for
+    // are less than 64 chars. We also don't care about encoding as those are always UTF.
+    // TODO: _MULTI_THREADED_EXECUTOR_
```

Comment:
decided to mark things that depend on the single-threaded executor from now on so we won't shoot ourselves in the foot if we ever gonna switch to a multi-threaded one 😅